app/models/hackbot/conversations/help.rb

```diff
@@ -0,0 +1,25 @@
+module Hackbot
+  module Conversations
+    class Help < Hackbot::Conversations::Channel
+      HELP_MENU = "Hi! I'm Hack Club's friendly robotic helper. I don't do "\
+                  "much right now, but if you're a club leader you've "\
+                  'probably received some messages from me about the '\
+                  'attendance of your club.'\
+                  "\n\n"\
+                  "If you're interested in contributing to my codebase check "\
+                  "out https://github.com/hackclub/api/issues. I'd love to "\
+                  'learn some new tricks from you!'.freeze
+
+      def self.should_start?(event, team)
+        Hackbot::Conversations::Mention.should_start?(event, team) &&
+          event[:text].include?('help')
```

Comment:
This is really clever, but probably simpler to just do `event[:text] =~ /hackbot help/`.

This might have some unintended side effects (ex. what happens if a user mentions "help hackbot"?).